simapp/app_test.go

```diff
@@ -310,7 +311,7 @@ func (c customAddressCodec) StringToBytes(text string) ([]byte, error) {
 }
 
 func (c customAddressCodec) BytesToString(bz []byte) (string, error) {
-	return string(bz), nil
+	return common.Bytes2Hex(bz), nil
```

Comment:
Wondering why didn't do `common.FromHex(text)` in `StringToBytes` method above ?

Reply:
Only fixed the `auth` tests and made a global replace for the address codec functions. So most probably I missed the `return []byte(text), nil` above.
Will definitely change that to `return common.FromHex(text), nil`, thanks!

Reply:
Pushed 🙌